cypress/support/pages/home.pageObject.js

```diff
@@ -3,6 +3,14 @@ import PageObject from '../PageObject';
 class HomePageObject extends PageObject {
   url = '/#/';
 
+  get visitMainPage() {
+    return cy.visit('http://localhost:3000');
+  }
```

Comment:
```suggestion
  get visitMainPage() {
    return cy.visit('http://localhost:3000');
  }
```
do not need to make getter here